core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -68,8 +68,7 @@ oppia.factory('SolutionObjectFactory', [
       var explanation = (
         $filter('convertToPlainText')(this.explanation));
       return (
-        '[' + solutionType + ' solution is ' + correctAnswer + '] ' +
-        explanation);
+        solutionType + ' solution is ' + correctAnswer + '. ' + explanation);
```

Comment:
The braces are around the whole thing; I'm talking only about correctAnswer.

In which specific cases do quotation marks occur? If those cases are reasonably common and look weird, then I retract my comment.